.teamcity/performance-test-durations.json

```diff
@@ -147,6 +147,18 @@
     "testProject" : "mediumMonolithicJavaProject",
     "linux" : 235000
   } ]
+}, {
+  "scenario" : "org.gradle.performance.regression.inception.BuildSrcApiChangePerformanceTest.buildSrc non-abi change",
```

Comment:
Not related to this change, rather my curiosity:
This test class used to test buildSrc changes on `gradle/gradle` build, thus `inception` in the original package name.
As the test evolved, the `gradle/gradle` inception part was removed and the test now only runs against generated test projects. Is there any reason to keep `inception` in the package name? Is this somehow related to how test results are stored in the performance db? Or are we free to change pakages as long as they are mirrored in these json configuration files?

Reply:
+1 for moving the test to another package. The fully qualified classname is stored in the performance DB, though I don't think it is a big issue to lose the history. And we can update the performance DB if we want to.